lua/nvim-treesitter/query.lua

```diff
@@ -12,7 +12,8 @@ end
 function M.get_query(ft, query_name)
   local query_files = api.nvim_get_runtime_file(string.format('queries/%s/%s.scm', ft, query_name), false)
   if #query_files > 0 then
-    return ts.parse_query(ft, read_query_file(query_files[1]))
+    local query = ts.parse_query(ft, read_query_file(query_files[1]))
+    return query
```

Comment:
why not returning immediately ?

Reply:
That's a mistake on my side,added some steps to help debugging bu forgot to preperly clean afterwards